azure-pipelines.yml

```diff
@@ -88,6 +102,8 @@ pool:
 jobs:
 - job: packageAndPublish
   displayName: Package And Publish Buyingcatalogue
+  # pool:
```

Comment:
delete please

Reply:
This was caused by merged changes into master after branch created. I have updated branch to reflect master